bee-node/Cargo.toml

```
@@ -36,9 +36,10 @@ iota-crypto = { version = "0.7.0", default-features = false, features = [ "ed255
 log = { version = "0.4.14", default-features = false }
 mime_guess = { version = "2.0.3", default-features = false, optional = true }
 multiaddr = { version = "0.13.0", default-features = false }
-paho-mqtt = { version = "0.9.1", default-features = false, features = [ "bundled" ]}
 rand = { version = "0.8.4", default-features = false }
 rpassword = { version = "5.0.1", default-features = false }
+rumqttd = { version = "0.8.0", default-features = false }
+rumqttlog = { version = "0.8.0", default-features = false }
```

There are already new versions of this crate out there.

```suggestion
rumqttd = { version = "0.9.0", default-features = false }
rumqttlog = { version = "0.9.0", default-features = false }
```